recipes/aiokafka/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "aiokafka" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 36a5aa5bd681455c59c0de8a0c1bb9cfd2b32d93481bf56e821e9bad24859415
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
```

Comment:
If c-extensions are being built, then you should have:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L37-L41

Are you choosing not to build the cython extensions? If you aren't going to do that, then you should remove `cython` from host. If you are, remove the `noarch: python` and add the c-compiler under `build`.

Reply:
should the dependency on a c compiler come transitively from the `cython` dependency?

Reply:
cython does not list a compiler as a `run` dependency:

https://github.com/conda-forge/cython-feedstock/blob/master/recipe/meta.yaml#L19-L27

Reply:
ok I've added it, would be better if conda understood that cython is itself a compiler and handled it appropriately but oh well

Reply:
You're certainly welcome to raise the issue on the cython feedstock, but I suspect that it has something to do with allowing the package to specify which c-compiler to use rather than having cython make that determination. 

Also if we are building C-extensions then you should remove `noarch: python`.